icekit/authors/models.py

```diff
@@ -0,0 +1,99 @@
+"""
+Model declaration for the `author` app.
+"""
+import re
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+from fluent_contents.extensions import PluginHtmlField
+from fluent_contents.models import PlaceholderField
+from icekit.articles.abstract_models import ListingPage, ArticleBase
+from icekit.validators import RelativeURLValidator
+
+
+class AuthorListing(ListingPage):
+    """
+    Author listing page to be mounted in fluent pages page tree.
+    """
+    class Meta:
+        verbose_name = 'Author listing'
+
+    def get_items(self):
+        """
+        :return: all published authors
+        """
+        return Author.objects.published()
+
+    def get_visible_items(self):
+        """
+        :return: all authors that can be viewed by the current user
+        """
+        return Author.objects.visible()
+
+
+class Author(ArticleBase):
```

Comment:
The CollectedContent is more generic a name, and defines no fields. Author no longer has `title`, but still uses a `parent` attribute to derive URL and eg for breadcrumbs.